src/main/java/dev/openfeature/sdk/Reason.java

```diff
@@ -1,5 +1,8 @@
 package dev.openfeature.sdk;
 
+/**
+ * Predefined resolution reasons.
+ */
```

Comment:
This one it debatable but I wanted to make it clear that you're not limited to just these reasons, these are just some we pre-defined.